cmd/crc/cmd/start.go

```diff
@@ -43,6 +44,16 @@ func runStart(arguments []string) {
 	if err := validateStartFlags(); err != nil {
 		errors.Exit(1)
 	}
+	newVersionAvailable, newVersion, err := version.NewVersionAvailable()
+	if err == nil && newVersionAvailable {
+		logging.Infof("A new version (%s) is available for download", newVersion)
+		yes := input.PromptUserForYesOrNo("Would you like to continue using the current version", false)
+		if !yes {
```

Comment:
I think we should change this logic, I mean it is confusing that using `!yes` and then asking to download new version. Also there should be a flag which disable this prompt (either in config or as start) ?

Reply:
variables named `yes` are hard to decipher, as you need to look at previous lines to give it meaning. Rather see it being named `hasNewVersionAvailable` or similarly.

Reply:
Renamed the variables

Reply:
@praveenkumar Added, config option and flag to start command, `--no-update-check`